<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectivityTest</class>
 <widget class="QMainWindow" name="ConnectivityTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConnectivityTest</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startTest">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="noLogin">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>50</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>No Login</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="outputToFile">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>80</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Output to File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="loginInput">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordInput">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="loginLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Login:</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="outputTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>611</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>774</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
